docker-compose.prod.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.7'
+services:
+  bot:
+    image: godfather:alpha
+    restart: "always"
+    command: "start"
```

Comment:
This will translate to `yarn start` in the docker, It takes the entrypoint and pastes whatever you put in command after that